base_layer/core/src/validation/mocks.rs

```diff
@@ -44,24 +47,24 @@ impl<T, B: BlockchainBackend> Validation<T, B> for MockValidator {
         }
     }
 }
-
-#[cfg(test)]
-mod test {
-    use crate::{
-        chain_storage::MemoryDatabase,
-        transactions::types::HashDigest,
-        validation::{mocks::MockValidator, Validation},
-    };
-
-    #[test]
-    fn mock_is_valid() {
-        let validator = MockValidator::new(true);
-        assert!(<MockValidator as Validation<_, MemoryDatabase<HashDigest>>>::validate(&validator, &()).is_ok());
-    }
-
-    #[test]
-    fn mock_is_invalid() {
-        let validator = MockValidator::new(false);
-        assert!(<MockValidator as Validation<_, MemoryDatabase<HashDigest>>>::validate(&validator, &()).is_err());
-    }
-}
+// #[cfg(test)]
```

Comment:
This test seems to test a very trivial case and didn't seem worth creating the full object graph to test something that is by definition correct